docs/Algemene Plaateslijke Verordening Tedeapolis.md

```diff
@@ -316,6 +316,20 @@ Het overtreden van het feit genoemd in lid 1 zal resulteren in een straf van de
 
 1. Bij een zelfmoord scenario moet de persoon in kwestie een account wipe krijgen en verplicht voor zijn nieuwe karakter een biografie schrijven. Mocht de persoon dit puur voor aandacht doen en niet uiteindelijk zijn/haar leven beëindigen, zal deze voor 200 uur naar de gevangenis gestuurd worden in isolatie.
 
+### Artikel 43 - Voertuigbelasting
```

Comment:
Ik wil alle kosten in een appart document hebben ik heb hier al een opzet voor, met dit er in nu missen er namelijk wat aspecten van het innen die niet er niet in staan

Reply:
Top, dan kan hij als niet geschreven beschouwd worden.